pdm/models/project_info.py

```diff
@@ -2,15 +2,21 @@
 
 import itertools
 import sys
-from typing import Any, Iterator
+from typing import Any, Iterator, List
 
 from pdm import termui
 from pdm.pep517.metadata import Metadata
 
 if sys.version_info >= (3, 8):
+    from email.message import Message as DistributionMetadata
     from importlib.metadata import Distribution
 else:
     from importlib_metadata import Distribution
+    from importlib_metadata._meta import PackageMetadata as DistributionMetadata
+
+
+def get_as_dot_list(metadata: DistributionMetadata, key: str) -> List[str]:
+    return metadata.get(key, "").split(",")
```

Comment:
Is this function necessary? It does the same as `keywords = metadata.get("keywords", "").replace(",", ", ")`, or just don't replace, use it as is.